superset-frontend/src/dashboard/components/nativeFilters/FilterBar/CrossFilters/CrossFilterTag.tsx

```diff
@@ -81,6 +84,7 @@ const CrossFilterTag = (props: {
       `}
       closable
       onClose={() => removeCrossFilter(filter.emitterId)}
+      closeIcon={customCloseIcon}
```

Comment:
All the time you have a closable Tag, the x should get a role button, so I believe it makes sense to move this within the `Tag` component